mothernet/prediction/mothernet_additive.py

```diff
@@ -72,11 +74,6 @@ def extract_additive_model(model, X_train, y_train, config=None, device="cpu", i
         else:
             output = model.transformer_encoder(train_x)
 
-        if model.marginal_residual in [True, 'True', 'output', 'decoder']:
```

Comment:
not sure if that was resolving a merge conflict but this seems to have gotten lost